examples/vnet/doca_dpdk_priv.h

```diff
@@ -1,21 +1,31 @@
 #ifndef _DOCA_DPDK_PRIV_H_
 #define _DOCA_DPDK_PRIV_H_
 
-struct doca_gw_port
-{
-    uint32_t port_id;
-    int      idx;
-    uint8_t  user_data[0];
-};
-
-
 struct doca_gw_pipelne_entry {
+	LIST_ENTRY(doca_gw_pipelne_entry) next;
+
     int id;
     void *pipe_entry;
     /* for deletion */
     int meter_id;
     int meter_profile_id;
 };
 
+struct doca_gw_pipeline {
+    void * handler;
+	uint32_t id;
+	uint32_t pipe_entry_id;
+	LIST_HEAD(, doca_gw_pipelne_entry) entry_list;
+};
+
+struct doca_gw_port
+{
+    uint32_t port_id;
+    int      idx;
+
+    struct doca_gw_pipeline *p_over_under;
+    struct doca_gw_pipeline *p_ol_ol;
+    uint8_t  user_data[0];
```

Comment:
just put it here a list of doca_gw_pipelines (you aleady put there list)

Reply:
done